README.md

```diff
@@ -1,21 +1,22 @@
 # mezzio-authorization-rbac
 
 [![Build Status](https://github.com/mezzio/mezzio-authorization-rbac/workflows/Continuous%20Integration/badge.svg)](https://github.com/mezzio/mezzio-authorization-rbac/actions?query=workflow%3A"Continuous+Integration")
+[![Type Coverage](https://shepherd.dev/github/mezzio/mezzio-authorization-rbac/coverage.svg)](https://shepherd.dev/github/mezzio/mezzio-authorization-rbac)
```

Comment:
Does this need adding on every repo?

Reply:
There's no mandate, but I don't see the harm in it.